### ISSUE TITLE
Trying to train on other dataset, but the result is not good

### ISSUE THREAD

**Issue by FCInter** [koyeongmin/PINet#12]
I manage to reimplement similar test result on Tusimple dataset using your code. Thank you very much!

But when I'm trying to train on my own dataset, the result is not very good. The output lane by the model has zigzag shapes which look very strange. I load the pretrained .pkl checkpoint you provided.

I tested learning rate 1e-4, 2e-4, batchsize 16, 32. The result does not change much.

Is there any other parameter that I can finetune?

Or is there something wrong with my implementation.

I did not change much to the code. Only add some lines to the preprocessing code, to format my dataset to the Tusimple-style, and I have double-checked that these changes did not cause anything wrong.

**Comment by koyeongmin** [comment]
The provided weight file was trained on the tuSimple dataset only, and this dataset is not general because it consists of only sunny day and highway images. So the file can not be good pretrained model.  I suggest to try to train from scratch. I am also trying to train the model on other dataset, Culane. It takes more time than tuSimple.

**Comment by SoulmateB** [comment]
hello,  do you train the model on CULane? how is the result？ I train on CULane，the result is not good. Is there any other parameter that I can finetune?

**Comment by koyeongmin** [comment]
I am also trying to train the model on CULane. When I get the result, I will update about it. Thank you!

**Comment by may545** [comment]
Could you share the result on Culane without training if available please? 
Thank you

**Comment by lucasjinreal** [comment]
@koyeongmin How's the result on CULane?

**Comment by koyeongmin** [comment]
https://github.com/koyeongmin/PINet_new 
This repository includes CULane result!
The concept is same, but networks and training parts are fixed a little bit.
